Include/Ashkal/Ashkal/Matrix.hpp

```diff
@@ -43,6 +43,32 @@ namespace Ashkal {
       std::array<std::array<double, 3>, 3> m_data;
   };
 
+  //! Returns a rotated identity Matrix.
+  /*
+    \param r The amount to rotate, in radians.
+  */
+  Matrix rotate(double r);
+
+  //! Returns a scaled identity Matrix.
+  /*
+    \param s The amount to scale, both x and y.
+  */
+  Matrix scale(double s);
+
+  //! Returns a scaled identity Matrix.
```

Comment:
```Returns a scaling matrix.```
```The horizontal scaling factor.```
```The vertical scaling factor.```

The reason is to avoid a connotation between scale and width/height. Some objects have no notion of width or height but can still scale horizontally or vertically.